.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Node.js CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ alpha, master ]
+  pull_request:
+    branches: [ alpha, master ]
+
```

Comment:
Could we change the coverage? 

I think this should only cover feature branches and the alpha branch. 

Note that the branches are as follows:

```
feature ->
             alpha -> develop -> master 
bugfix  ->
```

The reason it is like this is because of the fact that we currently have a stable fork on develop/master. 

I think we should change the test coverage to feature and alpha branch for now. Once these tests pass its very unlikely we will make changes (**WE SHOULDNT**) to the develop/master branches directly without having this code tested first.